examples/simple/lint/L-CEIE1-good.c

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) 2015 Runtime Verification, Inc. (RV-Match team). All Rights Reserved.
+
+#include <stdlib.h>
+#include <stdio.h>
+
+int main()
+{
+    int * number_ptr = malloc(sizeof(int));
+    *number_ptr = 10;
+    free(number_ptr);
+    printf("No memory Leaks, Execution Successful\n");
```

Comment:
Be uniform: either capitalize all words or none.  In fact, since the bad example was minimal, you can also make this one minimal (ie, remove assignment and printf).  Or did you have a reason for which you added the additional statements?